src/main/java/com/dev/cinema/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,48 @@
+package com.dev.cinema.dao.impl;
+
+import com.dev.cinema.dao.OrderDao;
+import com.dev.cinema.exception.DataProcessingException;
+import com.dev.cinema.lib.Dao;
+import com.dev.cinema.model.Order;
+import com.dev.cinema.model.User;
+import com.dev.cinema.util.HibernateUtil;
+import java.util.List;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+    @Override
+    public Order add(Order order) {
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.save(order);
+            transaction.commit();
+            return order;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can't add the order "
+                    + order + " to the DB.", e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+    }
+
+    @Override
+    public List<Order> getByUser(User user) {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            return session.createQuery("FROM Order o "
+                    + "JOIN FETCH o.user u "
```

Comment:
Do you need to fetch User since you are using ManyToOne for this field?

Reply:
?

Reply:
understood,  yes, I need use LEFT JOIN FETCH o.tickets t where o.user = :user" here